contrib/ci/Jenkinsfile.ampere

```diff
@@ -0,0 +1,131 @@
+#!groovy
```

Comment:
Is that a comment? What does it do?

Reply:
This indicates that the program is written in [groovy](https://en.wikipedia.org/wiki/Apache_Groovy) - its the configuration language for Jenkins.